dbt/adapters/base/impl.py

```diff
@@ -97,6 +97,7 @@
 GET_CATALOG_MACRO_NAME = "get_catalog"
 GET_CATALOG_RELATIONS_MACRO_NAME = "get_catalog_relations"
 FRESHNESS_MACRO_NAME = "collect_freshness"
+CUSTOME_SQL_FRESHNESS_MACRO_NAME = "collect_freshness_custom_sql"
```

Comment:
```suggestion
CUSTOM_SQL_FRESHNESS_MACRO_NAME = "collect_freshness_custom_sql"
```
typo CUSTOME->CUSTOM